package.json

```diff
@@ -1,17 +1,20 @@
 {
   "scripts": {
+    "mangata-rococo": "dotenv -e .env babel-node src/mangata-rococo.js",
     "mangata": "dotenv -e .env babel-node src/mangata.js",
     "setup-mangata": "dotenv -e .env babel-node src/setup-mangata.js",
     "shibuya": "dotenv -e .env babel-node src/shibuya.js",
     "lint": "eslint --fix src/*"
   },
   "dependencies": {
     "@inquirer/confirm": "^0.0.19-alpha.0",
-    "@mangata-finance/sdk": "^1.9.0",
+    "@mangata-finance/sdk": "^1.14.2",
```

Comment:
I upgraded mangata’s library and didn’t have any issue so far, so leave the changes here to upgrade to the latest version.